src/diracx/db/jobs/schema.py

```diff
@@ -198,3 +210,99 @@ class LoggingInfo(JobLoggingDBBase):
     StatusTimeOrder = Column(Numeric(precision=12, scale=3), default=0)
     StatusSource = Column(String(32), default="Unknown")
     __table_args__ = (PrimaryKeyConstraint("JobID", "SeqNum"),)
+
+
+class TaskQueues(TaskQueueDBBase):
```

Comment:
This is just stylistic, but: you added this in the same module where JobDB (and JobLoggingDB) is. SandboxMetadataDB is instead defined in another module. We either put all "WMS-related" schemas together or we split them in different files like it was in DIRAC.